### ISSUE TITLE
A global "Compositor Support" page?

### ISSUE THREAD

**Issue by AlanGriffiths** [vially/wayland-explorer#33]
Today I was asked about differences between compositors. In answering this I thought it would be useful for Wayland Explorer to have a global "Compositor Support" page covering all the extensions that have a "Compositor Support" section.

Is that feasible? Interesting?

**Comment by Conan-Kudo** [comment]
This would be great to have, because it's a little hard to figure out how each compositor stacks up to figure out the gaps unless you're drilling into specific protocols.

**Comment by VasanthBalguri** [comment]
Yes it is annoying to go through each protocol and trying to figure out which compositor is most supported or fits the requirements.

**Comment by Fuseteam** [comment]
i would like to help to add this, any pointers where i can look at to iterate?

**Comment by VasanthBalguri** [comment]
Hi,

On skimming the source I found that here the compositor data is present

https://github.com/vially/wayland-explorer/tree/main/src/data/compositors

Here is the component where it is rendered.

https://github.com/vially/wayland-explorer/blob/main/src/components/WaylandCompositors.tsx

Think it should be straightforward to create a new component. I have very
limited understanding of web development and am occupied in other projects.

**Comment by Fuseteam** [comment]
nice find, i'll try and look at it sometime
